packages/common/src/ui/icons/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
I've committed a suggestion cf400c3
Have taken the icon from [the closest I could find in zeplin](https://app.zeplin.io/project/60e44fff6fa942168092f1ff/screen/611ce286e8a19f15c3bc56c2)
and the colour from the [suggestion for the date picker icon](https://app.zeplin.io/project/60e44fff6fa942168092f1ff/screen/619a9538c40639a9f6a246cc)

I also thought it looked better smaller